src/ring_theory/ideal/operations.lean

```diff
@@ -845,6 +845,12 @@ lemma map_map {T : Type*} [ring T] {I : ideal R} (f : R →+* S)
 ((gc_map_comap f).compose _ _ _ _ (gc_map_comap g)).l_unique
   (gc_map_comap (g.comp f)) (λ _, comap_comap _ _)
 
+@[simp] lemma map_span (f : R →+* S) (s : set R) :
```

Comment:
Not entirely clear to me that this should be a `simp` lemma.

Reply:
I don't have a strong reason either way, and `submodule.map_span` is not a `simp` lemma, so I'll remove the attribute.